### ISSUE TITLE
Velocity js? 

### ISSUE THREAD

**Issue by weepy** [jsguy/mithril.animate#3]
You could easily add velocity js support?

**Comment by jsguy** [comment]
Yeah sure - I'm actually planning on doing that, as a fallback, at the moment it will use jquery animate, I just haven't gotten around to switching...
